docs/manual/kinds/ceos.md

```diff
@@ -204,6 +204,28 @@ cEOS Ma0 interface will be configured with a random MAC address with `00:1c:73`
 
 A default ipv4 route is also created with a next-hop of the management network to allow for outgoing connections.
 
+#### MGMT VRF
+The default empty configuration supports placing the management interface into a VRF to isolate it from the main device routing table.  Passing the environment variable `MGMT_VRF` in either the kind or node definition will activate this behavior, and alter the management services configuration to also reflect the management VRF.
+
+```yaml
+# example topo file with management VRF
+# node1 will have vrf MGMT
+# node2 will have vrf FOO
+name: ceos_vrf
+topology:
+  kinds:
+    ceos:
+      env:
+        MGMT_VRF: MGMT
```

Comment:
to avoid name collisions, lets use `CLAB_MGMT_VRF` env var name, to clearly say this is containerlab-significant env var.

most vars we introduce in clab follow this prefix principle

Reply:
Changed in latest changeset